### PR TITLE
Kontena-universal-lb addon

### PR DESCRIPTION
**Pull request by jnummelin**

### CODE REVIEW

File: non-oss/pharos_pro/addons/kontena-universal-lb/addon.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+Pharos.addon 'kontena-universal-lb' do
+  version '0.1.0'
+  license 'Kontena License'
```
Comment (jakolehm) [inline non-oss/pharos_pro/addons/kontena-universal-lb/addon.rb]: Might make sense to configure tolerations and node selectors? (can be follow-up pr)

File: non-oss/pharos_pro/addons/kontena-universal-lb/resources/02_operator.yml.erb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  replicas: 1
+  selector:
+    matchLabels:
+      name: akrobateo
+  template:
+    metadata:
+      labels:
+        name: akrobateo
+    spec:
+      serviceAccountName: akrobateo
```
[the remark this answers is not in the document]
Comment (jnummelin) [reply non-oss/pharos_pro/addons/kontena-universal-lb/resources/02_operator.yml.erb]: `kontena-cluster-critical`?

File: non-oss/pharos_pro/addons/kontena-universal-lb/addon.rb (hunk 1 above)
Comment (jnummelin) [reply non-oss/pharos_pro/addons/kontena-universal-lb/addon.rb]: I'd do it in a follow-up PR later on when we know how the LB daemonset will get their selectors & taints.

File: non-oss/pharos_pro/addons/kontena-universal-lb/resources/02_operator.yml.erb (hunk 2 above)
[the remark this answers is not in the document]
Comment (jakolehm) [reply non-oss/pharos_pro/addons/kontena-universal-lb/resources/02_operator.yml.erb]: > `kontena-cluster-critical`?

Yeah.

**Pull request merged**